src/consensus/blockchain.py

```diff
@@ -345,7 +345,7 @@ async def get_sp_and_ip_sub_slots(
         if block is None:
             return None
         is_overflow = self.sub_blocks[block.header_hash].overflow
-        if block is None:
+        if is_overflow is None:
```

Comment:
This is incorrect. we should remove the if statment altogether. Doing this in the other branch